internal/storage/fs/fixtures/fswithindex/fs.go

```diff
@@ -0,0 +1,6 @@
+package fswithindex
+
+import "embed"
+
+//go:embed *
+var FS embed.FS
```

Comment:
Quick cleanup suggestion on this:

You could put this embed statement directly in your `snapshot_test.go`.
If you put everything under `testdata` too, then this stuff gets skipped by the go tool.
(if you do this, then you will need to move this `embed` directive into your test)

Doing so would render this more like:
```
testdata
├── fswithindex
│   ├── prod
│   │   ├── features.yml
│   │   ├── prod.features.yml
│   │   └── prod.yml
│   ├── sandbox
│   │   ├── features.yaml
│   │   ├── sandbox.features.yaml
│   │   └── sandbox.yml
│   └── staging
│       ├── features.yml
│       ├── staging.features.yml
│       └── staging.yml
└── fswithoutindex
    ├── prod
    │   ├── features.yml
    │   ├── prod.features.yaml
    │   └── prod.yml
    ├── sandbox
    │   ├── features.yaml
    │   ├── sandbox.features.yml
    │   └── sandbox.yml
    └── staging
        ├── features.yml
        ├── staging.features.yaml
        └── staging.yml
```

In your test code you can use [fs.Sub](https://pkg.go.dev/io/fs#Sub) to turn sub directories into instances of `fs.FS` from your embed.FS for use in each test case.

For example:
```
//go:embed testdata/*
var testdata emded.FS

func Test_Someting(t *testing.T) {
	fs.Sub(testdata, "fswithindex")
}
```

Reply:
@GeorgeMac Much much cleaner. I appreciate the suggestion here.